### PR TITLE
D3ASIM-3447: Fixed REST endpoint creation.

### PR DESCRIPTION
**Pull request by hannesdiedrich**

### CODE REVIEW

File: d3a_api_client/aggregator.py [hunk 1]
```diff
@@ -64,6 +64,10 @@ def __init__(self, aggregator_name, simulation_id=None, domain_name=None,
         self.latest_grid_tree_flat = {}
         self.area_name_uuid_mapping = {}
 
+    @property
+    def endpoint_prefix(self):
```
Comment (Hassan754) [inline d3a_api_client/aggregator.py:68]: In the aggregator you can use the `d3a_api_client/rest_device.py:32` aggregator_prefix

Comment (hannesdiedrich) [reply d3a_api_client/aggregator.py:68]: NO! This is the actual bug! Please have a look at the differences between these two.

Comment (Hassan754) [reply d3a_api_client/aggregator.py:68]: This is the function that is returning the aggregator's prefix, it is the same except for the trailing slash 
```python
def get_aggregator_prefix(domain_name, simulation_id=None):
    return f"{domain_name}/external-connection/aggregator-api/{simulation_id}/"
```

Comment (hannesdiedrich) [reply d3a_api_client/aggregator.py:68]: Ah yes, now we are talking, but you referenced to `d3a_api_client/rest_device.py:32` not to `d3a_api_client/utils.py:78`

Comment (Hassan754) [reply d3a_api_client/aggregator.py:68]: It do call the get_aggregator_prefix from there

**Review by Hassan754 — APPROVED** [review]
LGTM

**Review by fievelk — APPROVED** [review]

**Pull request merged**